packages/sys-apps/cpuid/cpuid-20201006.exheres-0

```diff
@@ -17,12 +17,17 @@ MYOPTIONS="parts: binaries documentation"
 
 DEPENDENCIES=""
```

Comment:
I forgot to note this previous time: IIRC we don't usually include dependencies from `system` set, and `perl` (to run new script) is one of them.